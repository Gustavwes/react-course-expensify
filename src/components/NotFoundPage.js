import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => (
    <div>
        <h3>404 - Not found</h3>
        <Link to="/">Go home</Link>
    </div>
);


export default NotFoundPage;