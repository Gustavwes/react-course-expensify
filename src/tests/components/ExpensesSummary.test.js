import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('Should render ExpensesSummary correctly', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={25}/>);
    expect(wrapper).toMatchSnapshot();
});

test('Should render correctly with one expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={4} expensesTotal={2500}/>);
    expect(wrapper).toMatchSnapshot();
});