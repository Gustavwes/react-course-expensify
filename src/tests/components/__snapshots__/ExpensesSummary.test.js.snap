// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpensesSummary correctly 1`] = `
<div>
  <p>
    Viewing 
    1
     
    expense
     totalling 
    $0.25
  </p>
</div>
`;

exports[`Should render correctly with one expense 1`] = `
<div>
  <p>
    Viewing 
    4
     
    expenses
     totalling 
    $25.00
  </p>
</div>
`;
